bitstore/services.py

```diff
@@ -37,6 +43,21 @@ def verify(auth_token, owner):
         return False
 
 
+def permissions(auth_token):
```

Comment:
please use auth.verifyer for this

Reply:
Also no real need for _both_ verify and permissions, as they do the same thing...